Example/GiniVision/ComponentAPIReviewViewController.swift

```diff
@@ -63,6 +63,10 @@ class ComponentAPIReviewViewController: UIViewController {
         // Cancel analysis process to avoid unnecessary network calls.
         if parent == nil {
             AnalysisManager.sharedManager.cancelAnalysis()
+        } else {
+            if let nav = parent as? UINavigationController, let cameraContainer = nav.viewControllers[nav.viewControllers.count - 2] as? ComponentAPICameraViewController {
```

Comment:
That statement makes me really nervous. Especially the `nav.viewControllers[nav.viewControllers.count - 2]`.
Since this is in the sample app, which uses a storyboard, can't you use a rewind segue to pop to a specific view controller? I can help you with it, if you want.

Reply:
I tried to use the `unwind` segue so I could be more consistent with the segues, but I had to use a custom button for the UINavigationController (without back arrow) and add an IBAction in the viewController (when there was already a `didMoveTo...` method). 
What do you think?

Reply:
Even with a custom button you should be able to use unwind. Also, you can add the unwind segue as an outlet and perform it programmatically.

Reply:
Yes, I meant that maybe It could be overkill when we already have the `didMoveTo...` method, and we would also lose the _back arrow_ on back button.